pkg/cmd/repo/fork/fork.go

```diff
@@ -237,20 +239,14 @@ func forkRun(opts *ForkOptions) error {
 			}
 
 			if _, err := remotes.FindByName(remoteName); err == nil {
-				if connectedToTerminal {
-					return fmt.Errorf("a remote called '%s' already exists. You can rerun this command with --remote-name to specify a different remote name.", remoteName)
-				} else {
-					// TODO next major version we should break this behavior and force users to opt into
-					// remote renaming in a scripting context via --remote-name
-					renameTarget := "upstream"
-					renameCmd, err := git.GitCommand("remote", "rename", remoteName, renameTarget)
-					if err != nil {
-						return err
-					}
-					err = run.PrepareCmd(renameCmd).Run()
-					if err != nil {
-						return err
-					}
+				renameTarget := "upstream"
+				renameCmd, err := git.GitCommand("remote", "rename", remoteName, renameTarget)
```

Comment:
What happens here if `remoteName` is blank? If `remoteName` is blank should we use "origin"? I think that would mimic the behavior before #1882.

Reply:
this was my expectation as well, but we could notice and do a nicer error